cypress/e2e/court-cases/CourtCaseDetails.Notes.cy.ts

```diff
@@ -158,6 +182,114 @@ describe("Court case details", () => {
     clickTab("Notes")
     cy.contains("Case has no notes.")
   })
+
+  it("should not display the notes text area if the case is locked to another users", () => {
+    cy.task("insertCourtCasesWithNotesAndLock", {
+      caseNotes: [[]],
+      force: "02"
+    })
+    cy.login("bichard02@example.com", "password")
+    cy.visit("/bichard/court-cases/0")
+
+    clickTab("Notes")
+    cy.contains("Case has no notes.")
```

Comment:
This test doesn't seem to actually test that the notes text area isn't present. It might be worth using `cy.get(<notes text area selector>).should("not.exist")` to check this